test/runtests.jl

```diff
@@ -40,9 +56,10 @@ end
     using Base: divgcd
 
     function Base.:*(x::Fix{typeof(/), Tuple{N1, D1}}, y::Fix{typeof(/), Tuple{N2, D2}}) where {N1, D1, N2, D2}
```

Comment:
Can we replace the tests, which overload these super long and private signatures? (Seperate PR).

Reply:
I am not sure what "private signatures" you mean. What do you think could be improved with these tests?

Reply:
I did not understand, what these tests were supposed to test. To me, the type parameters of `Fix` were an implementation detail, that's what I meant by "private signatures". Let's discuss this below, where you explained the motivation of these tests.